ThemeDecorator/ThemeDecorator.js

```diff
@@ -203,7 +212,7 @@ const CustomizableSkinStyle = kind({
  */
 const ThemeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	// TODO: Document props passable to hoc ()
-	const {customSkin, float, i18n, noAutoFocus, overlay, ri, skin, spotlight} = config;
+	const {customSkin, float, i18n, noAutoFocus, overlay, ri, skin, spotlight, disableFullscreen} = config;
```

Comment:
It's not in alphabetical order, but I think it's not bad to put it at the end like Sandstone/Moonstone.
Please judge when merging